docker-store/publisher_faq.md

```diff
@@ -0,0 +1,144 @@
+*Installation instructions* is documentation on installing your software. In this case the documentation is just `Just launch the container and the map is going to be available on port 80 - ready-to-use - with instructions and list of available styles.` (We recommend more details for any content thats a certification candidate).
+
+#### How can I remove a submission?  I don’t want to currently have this image published as it is missing several information.
+If you would like your submission removed, let us know by contacting us at publisher-support@docker.com. We are redesigning our publisher portal, and will present this capability to you soon.
+ETA for ability to delete submissions that are not yet live is Q4 2017.
+
+#### Can publishers publish multi-container apps?
+Yes. Publishers can provide multiple images and add a compose file in the install instructions to describe how the multi-container app can be used.
+For now, we recommend asking publishers to look at this example from Microsoft https://store.docker.com/images/mssql-server-linux where they have Supported Tags listed in the Install instructions (you don't necessarily have to list it in the readme).
+
+#### Regarding source repo tags: it says not to use “latest”. However, if we want users to be able to download the images without specifying a tag, then presumably an image tagged “latest” is required. So how do we go about that?
+You can not submit "latest" tags via the certification/store publish workflow. The reason we do this is so that users are aware of the exact version they download. To make the user experience easy we have a copy widget that users can use to copy the pull command and paste in their command line. Here is a [screenshot](https://user-images.githubusercontent.com/2453622/32354702-1bec633a-bfe8-11e7-9f80-a02c26b1b10c.png) to provide additional clarity.
+
+#### I added a new repository via the publish flow, but Store displays older repositories. What gives?
+Go to `Plans and Pricing` section and update the repository in the `Plans and Pricing` page to the correct repository and resubmit. Here is a [screenshot](https://user-images.githubusercontent.com/2453622/32343986-85386882-bfc2-11e7-8afe-918056608f98.png) to add clarity.
+This is a UX issue on our side, and we plan on addressing it in Q4 '17.
+
+#### Can I submit more than 2 plans on Store?
+Not yet. ETA to present more than 2 plans is Q1 2018.
+
+#### Do you allow limited beta's or invite-only rollouts for products?
+All products will be discoverable on the Store, but we provide a means of invite-only rollouts ( Included in BYOL Documentation being addressed by Bryan Lee). Via the invite-only rollout only selected users can get entitled to content, but the product page will be discoverable by everyone)
```

Comment:
Remove space: ( Included in BYOL

Reply:
@gbarr01 I think we should remove this section entirely - it's a use case we'll formally support in the future, but for now, I think it will only cause more confusion.